src/submission/validation-clinical/surgery.ts

```diff
@@ -82,9 +82,19 @@ export const validate = async (
 		) as DeepReadonly<Therapy>;
 
 		if (existingSurgeryInDB || surgeries.length > 1) {
-			checkSurgeryDuplicateOrUpdate(therapyRecord, surgeries, existingSurgeryInDB, errors);
+			const duplicationError = checkSurgeryDuplicateOrUpdate(
+				therapyRecord,
+				surgeries,
+				existingSurgeryInDB,
+			);
+			if (duplicationError) errors.push(duplicationError);
```

Comment:
We want to always use brackets after an if statement.

- It keeps every logical block consistent which will help with the reader's ability to spot the start and end of the code block.
- It prevents future errors from editing mistakes. For example, where the single statement that is not wrapped in brackets is removed accidentally, making the next statement in the code now dependent on the code control statement before it.

Reply:
Updated